### PR TITLE
Add new para <SecureSocketsLayer>to Set-SPPeoplePickerConfig and Add-SPPeoplePickerSearchADDomain

### PR DESCRIPTION
**Pull request by xinyiHugeHard**

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9

File: sharepoint/sharepoint-server-ps/sharepoint-server/Add-SPPeoplePickerSearchADDomain.md [hunk 1]
```diff
@@ -131,6 +131,22 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -SecureSocketsLayer
+Whether the new added forest or domain should enable Secure LDAP (LDAPS).
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-server-ps/sharepoint-server/Add-SPPeoplePickerSearchADDomain.md:135]:
```suggestion
Specifies whether the newly added forest or domain should enable Secure LDAP (LDAPS).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by get-itips — APPROVED** [review]
I'm ok after @JohanFreelancer9 suggestions and also both SecureSocketsLayer descriptions should match

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by tiburd** [comment]
> I'm ok after @JohanFreelancer9 suggestions and also both SecureSocketsLayer descriptions should match

@xinyiHugeHard 
Please update to that the description for SecureSocketsLayer is the same in both files.
After you do that, please @ mention me.

Thank you

**Comment by xinyiHugeHard** [comment]
@tiburd Sorry for the late response, currently description of SecureSocketsLayer  in 2 files are both started with "Specifies whether " I don't think SecureSocketsLayer should have the same description in 2 files.

**Comment by tiburd** [comment]
> @tiburd Sorry for the late response, currently description of SecureSocketsLayer in 2 files are both started with "Specifies whether " I don't think SecureSocketsLayer should have the same description in 2 files.

@get-itips 
Hi Andres,
Can you please review my comment (in which I tried to repeat your comment) and Xinyi's response?
Thanks!

**Comment by get-itips** [comment]
I reviewed again and it is ok as it is as cmdlets are different @tiburd

**Pull request merged**